website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+### Inputs:
+
+-   Time step (Daily)
+-   Crop Type
+-   Pool 1: stem
+-   Pool 2: Leaves
+-   Pool 3: Storage Organs
+-   Pool 4: Roots
+-   Maximum and Minimum Temperature
+-   Global Radiation
+
+### Outputs:
+
+- It calculates how much carbon moves between pools.
+
+|![Pictorial representation of WOFOST Module.](../static/img/wofost-module-case-study.png) |
+|:--:|
+| Pictorial representation of WOFOST Module. |
+
+## Turnover Module
+
+Turnover is the movement from a living carbon pool to a dead carbon pool. This is due to the natural senescence of plant components . For example: A deciduous forest will have near 100 per cent turnover of leaves across a year – or an annual turnover rate of 0.99. The turnover rates will vary with vegetation types and the plant components. For example: A tree stem may have zero (0) annual turnover, while the leaves have one (1).
+
+To determine the volume of turnover through time, a calculation similar to the one below is carried out:
+
+```
+Turnover= 1 - (1 - turnover_rate) ^ time
+```
+
+Within the FLINT, the maximum decay rate is set at 0.99 999 999 998, and the minimum decay rate is 0.000001. These restrictions are in place to ensure there are no invalid calculations (Example: dividing by zero) occurs in the system.
+
+## Decomposition Module
+
+Decomposition represents heterotrophic respiration of litter and soil organic matter. The process of modelling decomposition is akin to turnover in plant material, where a proportion of biomass in litter and soil organic matter decays through time. The rates of this will vary with location based on the prevailing environmental conditions.
+
+## RothC Soil Carbon Module
+
+RothC is a module used for simulating the turnover of soil carbon in non-waterlogged soils under different vegetation types. It is based on the decomposition of organic matter as it relates to mean temperature and moisture, and proportion of resistant and decomposable organic material. It runs on the monthly time step interval.
+
+It is better than the other modules because its input data is readily available and there is considerable overlap with hybrid growth modules and WOFOST modules. For a detailed explanation and better understanding you can refer to the [RothC guide](https://www.verdeterreprod.fr/wp-content/uploads/2020/05/RothC_guide_WIN.pdf).
+
+### Inputs:
+
+-   Time Step (Monthly).
+-   Pool 1: Input of plant residue.
+-   Pool 2: Input of farmyard manure.
+-   Pool 3: Total organic carbon.
+-   Pool 4: Microbial biomass carbon.
+-   Rainfall.
+-   Open pan evaporation.
+-   Mean air temperature.
+-   Clay content of soil.
+-   Decomposability of incoming plant material.
+-   Soil Cover.
+-   Depth of soil layer sampled.
+
+### Output:
+
+-   Total Organic Carbon.
+-   Microbial Biomass Carbon.
+
+|![Pictorial representation of RothC Module.](../static/img/RothC-module-case-study.png) |
+|:--:|
+| Pictorial representation of RothC Module. |
```

Comment:
There are better diagrams to describe RothC. Take a look in this paper for example: https://www.researchgate.net/publication/43520343_Calibration_of_the_Rothamsted_organic_carbon_turnover_model_RothC_ver_263_using_measurable_soil_organic_carbon_pools